### ISSUE TITLE
[8.0/9.0/master] openerp-server: using multiple database names on db_name parameter will wrongly create a new database

### ISSUE THREAD

**Issue by aristobulo-eezee** [odoo/odoo#13079]
**Impacted versions:**
8.0, 9.0, master (maybe older versions but not relevant)

**Steps to reproduce:**

1. setup a new odoo instance
2. run odoo like this:

`./openerp-server -r odoo -d db1,db2 --workers 2 --db-filter db.*$`

Please notice that for this issue what's important is passing multiple database names to `-d` parameter or into `db_name` on a odoo configuration file.

**Current behavior:**

1. odoo server will create a database with name `db1,db2`

2. If `db1` and `db2` exist and are not initialized, odoo will run `init_db` on `db1` and `db2` and will create `db1,db2` and leave it un-initialized

3. If `db1` and `db2` do not exist odoo will create database `db1,db2` and will run `init_db` on it. Also it will raise 

```OperationalError: FATAL:  database "db1" does not exist``` and 

```OperationalError: FATAL:  database "db2" does not exist```

**Expected behavior:**

Odoo should not create `db1,db2` database.

Optional (?): Odoo server should start and create only those not existing databases included sent to `-d` parameter.

**Explanation:**

In the scenario where you are running one instance of odoo with multiple databases the use of `db_name` or `-d` parameter is useful to avoid `CronWorkers` to create connections to undesired databases. (Combined with `db_filter` option to display desired databases)

I'm assuming that comma-separated list of database names (i.e: 'db1,db2,db3') are supported because of:

https://github.com/odoo/odoo/blob/9.0/openerp/cli/server.py#L155

and 

https://github.com/odoo/odoo/blob/master/openerp/service/server.py#L784

Basically, the described issue is caused here:

https://github.com/odoo/odoo/blob/9.0/openerp/cli/start.py#L55

as `db_name` is not being split before evaluation, like it's done on `openerp.cli.server` and `openerp.service.server`

**Comment by aristobulo-eezee** [comment]
@amigrave Could you take a look on this, please?

Hi @antonylesuisse, I found this while investigating a bit more about the issue I talked to you before, regarding the strange behavior of some workers closing connections from other instances. Using `db_name` is a workaround I'm testing because like you pointed, `db_name` is used by the worker to retrieve the list of available databases and using `db_filter` does nothing to the worker.

**Comment by antonylesuisse** [comment]
Actually it's https://github.com/odoo/odoo/blob/9.0/openerp/cli/server.py#L133 that needs a for on the splitted db_name. Start is an other command that only supports one database.

**Comment by aristobulo-eezee** [comment]
@antonylesuisse thanks for the info. I will check that

**Comment by aristobulo-eezee** [comment]
@antonylesuisse at https://github.com/odoo/odoo/blob/9.0/openerp/cli/server.py#L155 it's done  and saved into `preload`.

What if?

[db_name.txt](https://github.com/odoo/odoo/files/409045/db_name.txt)

**Comment by odony** [comment]
@aristobulo-eezee Would you like to make a PR with this small patch on 9.0, so we can credit it to you? It looks good on first look.
Also, if I remember correctly this issue does not affect Odoo 8.0, as `-d` did not automatically create databases, only the `start` command did it, and it only support one DB.

**Comment by aristobulo-eezee** [comment]
@odony ok, I will create the PR

**Comment by RickHunterEc** [comment]
Hi, @odony, same problem in master, but not PR merged.
